internal/awslambda/extension/manager.go

```diff
@@ -214,6 +230,41 @@ func (m *manager) register(ctx context.Context) error {
 	return nil
 }
 
+func (m *manager) subscribeToTelemetry(ctx context.Context) error {
+	b, err := jsoniter.Marshal(telemetry.SubscriptionRequest{
+		SchemaVersion: telemetry.ApiVersion,
+		Types:         []string{telemetry.PlatformSubscriptionType},
+		Destination:   &telemetry.SubscriptionDestination{URI: m.telemetryServer.Endpoint(), Protocol: telemetry.ProtocolHTTP},
+		// we buffer for the minimum time on the lambda side to avoid adding additional billedDuration to the lambda
+		Buffering: &telemetry.SubscriptionBufferingConfig{TimeoutMs: telemetry.MinBufferingTimeoutMs},
+	})
+	if err != nil {
+		return err
+	}
+	req, err := http.NewRequestWithContext(ctx, http.MethodPut, telemetry.SubscribeEndpoint.GetUrl(m.domain), bytes.NewReader(b))
+	req.Header.Set(api.LambdaExtensionIdentifierHeaderKey, m.registeredID)
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := m.client.Do(req)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		_, _ = io.Copy(io.Discard, resp.Body)
+		_ = resp.Body.Close()
+	}()
+
+	if resp.StatusCode != http.StatusOK {
+		body, _ := io.ReadAll(resp.Body)
+		m.log.WithError(err).Errorf("received %d response code from telemetry subscription with body %s", resp.StatusCode, string(body))
+		return ErrFailedTelemetrySubscription
+	}
+
+	m.log.Info("successfully subscribed to telemetry API")
```

Comment:
```suggestion
	m.log.Info("Successfully subscribed to telemetry API")
```

Ensuring that log lines begin with an Upper case.